app/api/helpers/tasks.py

```diff
@@ -452,3 +452,12 @@ def export_speakers_pdf_task(self, event_id):
         result = {'__error': True, 'result': str(e)}
 
     return result
+
+
+@celery.task(base=RequestContextTask, name='delete.translations', bind=True)
+def delete_translations(self, zip_filename, uuid_literal, temp_dir_path):
+    try:
+        os.remove(os.path.join(temp_dir_path, zip_filename))
+    except Exception as e:
+        result = {'__error': True, 'result': str(e)}
+        logging.error(traceback.format_exc())
```

Comment:
Again. This should be a human readable string and not the exception message

Reply:
Could you suggest how I can obtain the human readable message?